packages/text-field/test/visual/lumo/vaadin-text-field.js

```diff
@@ -0,0 +1,8 @@
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
```

Comment:
Should this file be also removed in https://github.com/vaadin/web-components/issues/2220?

Reply:
Done.